compiler/tflchef/tflite/src/Op/BidirectionalSequenceLSTM.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "BidirectionalSequenceLSTM.h"
+
+#include "Convert.h"
+#include "FillerHelper.h"
+
+namespace tflchef
+{
+
+void TFliteOpBidirectionalSequenceLSTM::filler(const tflite::Operator *op, TFliteImport *import,
+                                               tflchef::ModelRecipe *model_recipe) const
+{
+  const std::vector<int32_t> &inputs = as_index_vector(op->inputs());
+  assert(inputs.size() == 48);
+
+  for (int32_t i = 0; i < inputs.size(); i++)
+  {
+    // Except for Input 0, 17, 34, 35 and 37.
+    // Each Input mean  Input Tensor, OutputState Tensor and CellState Tensor.
+    // This could be updated from previous input or User Given data, so This could not be Const
+    if (i == 0 || i == 17 || i == 34 || i == 35 || i == 37)
```

Comment:
@seanshpark , @mhs4670go , could you please comment on whether this looks alright?

Reply:
@struss PTAL

Reply:
I think I do some mistake while createing `UniDirectionalSequenceLSTM` and this make some effect on this. I think User input tensor and state tensor be `0`, `35`,  `36`,  `37`,  `38` on this. and also, auxInput(39) is optional for this but give same as input tensor on tflite kernel test example. I think it could be handled as same with input.

Reply:
> I think I do some mistake while createing UniDirectionalSequenceLSTM and this make some effect on this.

Maybe revise `UniDirectionalSequenceLSTM` ?